pubspec.yaml

```diff
@@ -43,3 +43,16 @@ dev_dependencies:
 
 executables:
   pana:
+
+dependency_overrides:
```

Comment:
Does this mean we can't release pana for a while if this gets merged?

Reply:
We can add a git override in `pub-dev` for `pana` and these two dependencies too. But yes, that's painful.

Reply:
Yes unfortunately it will mean that - I am trying to ping for a new analyzer version here: https://github.com/dart-lang/sdk/issues/37487